src/App.js

```diff
@@ -1,13 +1,28 @@
 import BookList from './views/Books/BookList';
+import BookDetail from './views/Books/BookDetail';
 import './App.css';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
 
 function App() {
-  // TODO: Add routes to books & views
   return (
-    <main className="container">
-      <h1>Library Catalog</h1>
-      <BookList />
-    </main>
+    <BrowserRouter>
+      <main className="container">
+        <h1>Library Catalog</h1>
+        <Switch>
+          <Route exact path='/'>
+            <p>check out our sick-ass <a href='/books'>books</a></p>
```

Comment:
Using an <a> tag will reload the page and thus mean you have to reload all your react code -- a Link / NavLink will just change the url bar and switch out your component